tinyagentos/routes/skill_exec.py

```
@@ -225,6 +225,16 @@ async def _skill_notify_user(args: dict, request: Request) -> dict:
         return {"error": str(exc)}
 
 
+async def _skill_list_projects(args: dict, request: Request) -> dict:
+    """List the user's projects so the agent can pick one."""
+    try:
+        from tinyagentos.tools.project_tools import execute_list_projects
+
+        return await execute_list_projects(args, request)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)